src/Components/Character/CharacterInfo.css

```diff
@@ -0,0 +1,31 @@
+#characterInfo {
+  width: 400px;
+  height: 100px;
+  background-color: darkblue;
+  position: absolute;
+  text-align: center;
+  clip-path: polygon(0 20px, 20px 0, 400px 0, 360px 60px, 180px 60px, 160px 100px, 40px 100px);
+  z-index: 4;
```

Comment:
why `z-index: 2` is not enough?

Reply:
Because it layers in relation with both the exp components, and i want the to stack properly